librz/util/vector.c

```diff
@@ -85,23 +85,27 @@ RZ_API void rz_vector_free(RzVector *vec) {
  * \param src The vector to clone from.
  * \return true on success, false on failure.
  */
-RZ_API bool rz_vector_clone_into(
+RZ_API bool rz_vector_clone_intof(
```

Comment:
Please also keep the original functions as well, you removed them, also update Doxygen